models/hds.go

```diff
@@ -320,7 +320,7 @@ var hqmf_qrda_oids = []byte(`[
         "attribute": "",
         "exclude_null_flavor": false,
         "extra_content": "xsi:type=\"CD\" sdtc:valueSet=\"{{.MapDataCriteria.ValueSetOid}}\"",
-        "required_code_sets": ["2.16.840.1.113883.3.88.12.3221.7.4"],
```

Comment:
Also a change requested by @dczulada so @sylinmino, he will do a better job than I explaining why this change.